Readme.md

```diff
@@ -373,6 +373,33 @@ OPTIONS:
             Use as `--rustc-extra-args="--my-arg"`
 ```
 
+### Upload
+
+```
+Uploads the distribution files to the repository (package index)
+
+It behaves like `twine upload`, but is restricted to the python wheels and source distributions.
```

Comment:
I wouldn't be so bold to claim that it has the same behaviour as `twine`. There are a lot of edge cases maturin doesn't (and will never) cover.

Reply:
I've reworded it to look more like the original comment.